Build warnings when using CMAKE_DEBUG_POSTFIX for the crasphad_wer.dll

I am using cmake (3.31.6-msvc6 the version provided in combination with VS2022) to build sentry-native.
We are using sentry release/0.12.0 branch from 29.Oct.2025 (a26685c742470e0c0f6a9b97aa5c321c2b9cbbe2)

Our libraries in debug and release are provided side by side with a suffix d.
sentry-native.lib
sentry-natived.lib

cmake offers an option for exactly this: CMAKE_DEBUG_POSTFIX="d"

The command line we use for cmake:
```
cmake -B buildwin32_143 -DSENTRY_BACKEND=crashpad -DSENTRY_BUILD_TESTS=OFF -DSENTRY_BUILD_EXAMPLES=OFF -DSENTRY_INTEGRATION_QT=OFF -DSENTRY_BUILD_SHARED_LIBS=OFF -DSENTRY_BUILD_RUNTIMESTATIC=ON -DCMAKE_DEBUG_POSTFIX="d" -DCMAKE_INSTALL_PREFIX=. -DCMAKE_INSTALL_LIBDIR=../lib_143 -DCMAKE_INSTALL_BINDIR=../lib_143 -A Win32 -G "Visual Studio 17 2022" 1>somelogfolder/cmake_sentry_native_win32_143.log 2>somelogfolder/cmake_sentry_native_win32_143.err
```

Using this option i get compile errors for the crasphad_werd.dll when building the created vcxproject files:

```
crashpad_werd.exp : warning LNK4070: /OUT:crashpad_wer.dll directive in .EXP differs from output filename 'X:\dev\global\libs\sentry-native\buildx64_vc143\crashpad_build\handler\Debug\crashpad_werd.dll'; ignoring directive [X:\dev\global\libs\sentry-native\buildx64_vc143\crashpad_build\handler\crashpad_wer.vcxproj]
```

I´ve raised this in a discussion with another cmake build error we had before and @supervacuus suggested to create a dedicated ticket which i am now doing...
https://github.com/getsentry/sentry-native/discussions/1380

--- a/handler/CMakeLists.txt
+++ b/handler/CMakeLists.txt
@@ -147,6 +147,7 @@ if (WIN32)
 
     set_property(TARGET crashpad_wer PROPERTY EXPORT_NAME crashpad_wer)
     set_property(TARGET crashpad_wer PROPERTY PREFIX "") # ensure MINGW doesn't prefix "lib" to dll name
+    set_property(TARGET crashpad_wer PROPERTY DEBUG_POSTFIX "") # prevent CMAKE_DEBUG_POSTFIX from being applied
     add_library(crashpad::wer ALIAS crashpad_wer)
 
     if (COMMAND sentry_add_version_resource)

